src/app/repos/_components/UserCard.tsx

```diff
@@ -0,0 +1,77 @@
+'use client';
+
+import Image from 'next/image';
+import Link from 'next/link';
+import { useRouter } from 'next/navigation';
+import { twMerge } from 'tailwind-merge';
+
+import Button from '@/components/Button/Button';
+import { Modal } from '@/components/Modals';
+import { useModal } from '@/hooks/useModal';
+import { RightArrowIcon, SignOutIcon } from '@/public/index';
+
+type UserCardProps = {
+  avatar: string;
+  email: string;
+  hasLogoutButton?: boolean;
+};
+
+export default function UserCard({ avatar, email, hasLogoutButton }: UserCardProps) {
+  const router = useRouter();
+  const [isModalOpen, handleClickTrigger] = useModal();
+
+  const handleLogout = () => {
+    handleClickTrigger();
+    router.push('/');
+  };
+
+  const baseStyles = 'flex w-full items-center justify-between ';
+
+  const logoutStyles = 'border-b border-b-[#e6e6e6]] pb-[8rem]';
+  const defaultStyles = 'bg-neutral-5 rounded-[4.2rem] p-[3.2rem]';
+
+  return (
+    <section className={twMerge(baseStyles, hasLogoutButton ? logoutStyles : defaultStyles)}>
+      <div className="flex items-center gap-[4.4rem]">
+        <Image src={avatar} alt="avatar" width={107} height={107} />
```

Comment:
얘는 더미가 동그래서 지금은 티가 안나는데 네모난 이미지로 올거라서
겉을 감싸고, overflow-hidden, rounded-full 해줘야 할거 같아용
제안은 참고만 해주세용 
```suggestion
<div className = 'overflow-hidden rounded-full w-[10.7rem] h-[10.7rem] relative '>
        <Image src={avatar} alt="avatar" fill/>
</div>
```